### PR TITLE
feat: filter-added

### PR DESCRIPTION
**Pull request by Megha-Joshi**
In this PR, I have added filtering of video cards according to their categories.

### CODE REVIEW

**Review by HeyNitin — APPROVED** [review]
The code looks good, 
Minor suggestions: You can use absolute imports and please add screenshots or preview deploy in PRs

File: src/context/filterContext.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                const filterArr = showVideos.filter((item) => item.category === filterOnCategory)
+                return filterArr;
+            }
+            return showVideos;
+        }
+    }
+
+    const filteredVideos = (videos, filterState) => {
+        const { filterByCategory } = filterState;
+        console.log(filterByCategory);
```
Comment (shrushti2000) [inline src/context/filterContext.js:45]: can remove this console log

**Review by shrushti2000 — APPROVED** [review]
Everything looks good.
Proper destructuring wherever possible  and made use of hooks as well.

File: src/context/filterContext.js (hunk 1 above)
Comment (Megha-Joshi) [reply src/context/filterContext.js:45]: Okay

**Pull request merged**